.flaskenv

```diff
@@ -0,0 +1,8 @@
+DM_API_PORT=5000
+DM_SEARCH_API_PORT=5009
+DM_BUYER_PORT=5002
+
+FLASK_APP=application:application
+FLASK_ENV=development
+FLASK_RUN_EXTRA_FILES=app/content/frameworks/
+FLASK_RUN_PORT=${DM_BUYER_PORT}
```

Comment:
Should `.flaskenv` be included in `.dockerignore`?

Reply:
Good idea

Reply:
It shouldn't be a problem if it is included, but better safe than sorry

Reply:
Ah, I remember now, our `.dockerignore` files for our apps exclude everything then add only certain files; we don't want this file to be included in the Docker image, so we don't add it to the `.dockerignore` file.

Reply:
Ahh that's right. Well remembered!